requirements.txt

```diff
@@ -1,12 +1,15 @@
 requests
 future
-astropy
+astropy>=3.2
```

Comment:
maybe is it good to set it like in the dispatcher ?

https://github.com/oda-hub/dispatcher-app/blob/6e98f24012ab351d73a83801a42aafedfd3b202c/requirements.txt#L6

Reply:
Done also in setup.py